packages/medusa/src/migrations/1672906846559-category.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class category1672906846559 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      CREATE TABLE "category"
+        (
+          "id" character varying NOT NULL,
+          "name" text NOT NULL,
+          "handle" text NOT NULL,
+          "parent_category_id" character varying,
```

Comment:
suggestion(non blocking): should it be nullable for top level?

Reply:
default is nullable, right?